orchestra/static/orchestra/common/css/lib/style.css

```diff
@@ -248,27 +248,6 @@ ul.top-menu > li > .logout {
     padding-bottom: 10px;
 }
 
-/*LEFT NAVIGATION ICON*/

```

Comment:
note: these image files are missing and fail the whitenoise build. the class is unused, @derek-schultz says it's ok to remove